BackEnd/src/main/java/com/exadel/carpoolfree/model/Path.java

```diff
@@ -17,5 +22,6 @@
     @GeneratedValue(strategy = GenerationType.IDENTITY)
     private Long id;
 
+    @Column
     private String coordinates;

```

Comment:
It's fine for now but we might consider a better format for this when we know better what we get from yandex